.travis.yml

```diff
@@ -28,7 +28,8 @@ script:
 - nosetests -v TestJunOSDriver:TestGetterJunOSDriver.test_get_lldp_neighbors
 - nosetests -v TestJunOSDriver:TestGetterJunOSDriver.test_get_lldp_neighbors_detail
 - nosetests -v TestJunOSDriver:TestGetterJunOSDriver.test_get_arp_table
-- nosetests -v TestJunOSDriver:TestGetterJunOSDriver.test_get_ntp_peers
+- nosetests -v TestJunOSDriver:TestGetterJunOSDriver.test_get_ntp_stats
+#- nosetests -v TestJunOSDriver:TestGetterJunOSDriver.test_get_ntp_stats
```

Comment:
Just remove those lines. The method ``get_ntp_peers`` doesn't exist. If we implement it later on we can add this line again.